client/state/comments/selectors.js

```diff
@@ -47,8 +43,9 @@ export const getPostTotalCommentsCount = ( state, siteId, postId ) => state.comm
  */
 export const getPostMostRecentCommentDate = createSelector(
 	( state, siteId, postId ) => {
-		const items = getPostCommentItems( state, siteId, postId );
-		return items && items.first() ? new Date( items.first().get( 'date' ) ) : undefined;
+		const items = filter( getPostCommentItems( state, siteId, postId ), { parent: false } );
```

Comment:
Same as earlier, this wasn't trusting the order of the items on the store. Fixed on 18bc439ab4db1a1d243a4901d426fcb5f5fec705.